src/main/java/org/broadinstitute/consent/http/models/sam/UserStatus.java

```diff
@@ -0,0 +1,89 @@
+package org.broadinstitute.consent.http.models.sam;
+
+import com.google.gson.Gson;
+
+/** This represents the Sam response when a new user is created via POST /register/user/v2/self */
+public class UserStatus {
+
+  private UserInfo userInfo;
+  private Enabled enabled;
+
+  public UserInfo getUserInfo() {
+    return userInfo;
+  }
+
+  public UserStatus setUserInfo(UserInfo userInfo) {
+    this.userInfo = userInfo;
+    return this;
+  }
+
+  public Enabled getEnabled() {
+    return enabled;
+  }
+
+  public UserStatus setEnabled(Enabled enabled) {
+    this.enabled = enabled;
+    return this;
+  }
+
+  @Override
+  public String toString() {
+    return new Gson().toJson(this);
+  }
+
+  public static class UserInfo {
+    private String userSubjectId;
+    private String userEmail;
+
+    public String getUserSubjectId() {
+      return userSubjectId;
+    }
+
+    public UserInfo setUserSubjectId(String userSubjectId) {
+      this.userSubjectId = userSubjectId;
+      return this;
+    }
+
+    public String getUserEmail() {
+      return userEmail;
+    }
+
+    public UserInfo setUserEmail(String userEmail) {
+      this.userEmail = userEmail;
+      return this;
+    }
+  }
+
+  public static class Enabled {
+    private Boolean ldap;
```

Comment:
What does `ldap` represent?

Reply:
I think it means there is a user saved in our ldap server. Sam uses another subsystem, OpenDJ, which I think is an ldap instance. https://github.com/broadinstitute/firecloud-opendj